published/src/test/java/com/google/spark/bigquery/acceptance/AcceptanceTestUtils.java

```diff
@@ -125,12 +137,65 @@ protected static void deleteGcsDir(String testBaseGcsDir) throws Exception {
     BlobId[] blobIds =
         StreamSupport.stream(
                 storage
-                    .list(uri.getHost(), Storage.BlobListOption.prefix(uri.getPath().substring(1)))
+                    .list(
+                        uri.getAuthority(),
+                        Storage.BlobListOption.prefix(uri.getPath().substring(1)))
                     .iterateAll()
                     .spliterator(),
                 false)
             .map(Blob::getBlobId)
             .toArray(BlobId[]::new);
     storage.delete(blobIds);
   }
+
+  public static void createBqDataset(String dataset) {
+    DatasetId datasetId = DatasetId.of(dataset);
+    BigQueryOptions.getDefaultInstance().getService().create(DatasetInfo.of(datasetId));
```

Comment:
Please create a private static member `bigquery` which is initialized once

Reply:
Cannot initialise the dataset once, as every acceptance calls its own "setup" and "tearDown".

Reply:
The `bigquery` member should be initialized once.

Reply:
done